AsyncDisplayKit/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -354,6 +354,34 @@ - (void)setNeedsLayout
   }
 }
 
+- (void)layoutIfNeeded
```

Comment:
I think there are some complexities to adding this - because a "bridged" version wouldn't be synchronous anyway, I'm not sure it is needed compared to setNeedsLayout.

I trust your judgement if you want to leave this in -- but I think we should remove it from the property bridge .h and .m for right now and bring it back in a later diff.